src/coreclr/nativeaot/Runtime/PalRedhawkCommon.h

```diff
@@ -91,6 +91,37 @@ struct PAL_LIMITED_CONTEXT
     uintptr_t GetLr() const { return LR; }
     void SetIp(uintptr_t ip) { IP = ip; }
     void SetSp(uintptr_t sp) { SP = sp; }
+#elif defined(TARGET_LOONGARCH64)
+    uintptr_t  FP;
+    uintptr_t  RA;
+
+    uintptr_t  R4;
+    uintptr_t  R5;
+    uintptr_t  R23;
+    uintptr_t  R24;
+    uintptr_t  R25;
+    uintptr_t  R26;
+    uintptr_t  R27;
+    uintptr_t  R28;
+    uintptr_t  R29;
+    uintptr_t  R30;
+    uintptr_t  R31;
+    uintptr_t  R2;
+
+    uintptr_t  SP;
+    uintptr_t  IP;
+
+    uint64_t      F[32 - 24]; // Only the F registers F24..F31 need to be preserved
```

Comment:
Should this be `4 *` as well?

Reply:
I think here only save the float callee saved registers F24-F31, there is no need save the SIMD vector registers.

Reply:
The arm64 also only saved the float callee saved registers.

Reply:
I'm not sure if it should be `4 *` in `PAL_LIMITED_CONTEXT`. I thought it might be consistent with `REGDISPLAY`.

Reply:
I think the question is what Loongarch psABI says about caller/callee saved registers and how it was amended for LSX/LASX.

Reply:
yes, you are right.
For LoongArch64, the float callee saved regitsters only save the low 64bits, which same with arm64
https://github.com/dotnet/runtime/blob/dd6840d5e579b4a8a77814f0b96e5ace56da4932/src/coreclr/nativeaot/Runtime/PalRedhawkCommon.h#L84-L85

https://github.com/dotnet/runtime/blob/dd6840d5e579b4a8a77814f0b96e5ace56da4932/src/coreclr/nativeaot/Runtime/arm64/ExceptionHandling.S#L30-L33

The LoongArch64 is also only saved the low 64bits here
https://github.com/dotnet/runtime/blob/dd6840d5e579b4a8a77814f0b96e5ace56da4932/src/coreclr/nativeaot/Runtime/loongarch64/ExceptionHandling.S#L32-L39